receiver/kafkametricsreceiver/metadata.yaml

```
@@ -18,17 +18,125 @@ attributes:
   group:
     description: The ID (string) of a consumer group
     type: string
+  broker:
+    description: The ID (integer) of a broker
+    type: int
 
 metrics:
+
   #  brokers scraper
   kafka.brokers:
     enabled: true
+    description: "[DEPRECATED] Number of brokers in the cluster."
+    unit: "{brokers}"
+    gauge:
+      value_type: int
+    warnings:
+      if_enabled: The metric is deprecated and will be removed. Use `messaging.kafka.broker.count`
+
+  messaging.kafka.broker.count:
+    enabled: false
     description: Number of brokers in the cluster.
     unit: "{brokers}"
     sum:
       monotonic: false
       value_type: int
       aggregation_temporality: cumulative
+    warnings:
+      if_enabled_not_set: This metric will be enabled by default in the next versions.
+  messaging.kafka.broker.consumer_fetch_rate:
+    enabled: false
+    description: Average consumer fetch Rate
+    unit: "{fetches}/s"
+    gauge:
+      value_type: double
+    attributes: [broker]
+    warnings:
+      if_enabled_not_set: This metric will be enabled by default in the next versions.
+  messaging.kafka.broker.incoming_byte_rate:
+    enabled: false
+    description: Average tncoming Byte Rate in bytes/second
+    unit: 1
+    gauge:
+      value_type: double
+    attributes: [broker]
+    warnings:
+      if_enabled_not_set: This metric will be enabled by default in the next versions.
+  messaging.kafka.broker.outgoing_byte_rate:
+    enabled: false
+    description: Average outgoing Byte Rate in bytes/second.
+    unit: 1
+    gauge:
+      value_type: double
+    attributes: [broker]
+    warnings:
+      if_enabled_not_set: This metric will be enabled by default in the next versions.
+  messaging.kafka.broker.request_latency:
+    enabled: false
+    description: Average request latency in ms
+    unit: "ms"
```

Should be in seconds https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-units
```suggestion
    unit: "s"
```